File: 7.data deletion.py
# // write a program to delete an element from an array

import numpy as np
arr = np.zeros(50,int)
n = int(input("enter the number of elements of the array "))
print("enter elements ")
for i in range(0,n):
    arr[i] = int(input())

pos = int(input("enter the position to delete "))
if(pos>n):
	print("invalid position")
else:
    for i in range(pos-1,n):
        arr[i] = arr[i+1]

    print("the array is")
    for i in range(0,n-1):
        print(arr[i])

File: 7.data deletion.c
// write a program to delete an element from an array

#include<stdio.h>
int main(){
	int arr[10],pos,n;
	
	printf("enter the number of elemetns of the array: \n");
	scanf("%d",&n);
	
	
	printf("enter the elements: \n");
	
	for(int i = 0;i<n;i++)
		scanf("%d",&arr[i]);
		
	printf("enter the position to delete: \n");
	scanf("%d",&pos);
	
	for(int i = pos-1;i<n;i++)
		arr[i] = arr[i+1];
		
	printf("the array is\n");
	for(int i = 0;i<n-1;i++)
		printf("%d ",arr[i]);
return 0;
}